email_template_qweb/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "QWeb for email templates",
+    "version": "12.0.1.0.0",
+    "author": "Therp BV, "
+              "Tecnativa, "
```

Comment:
you can't claim being an author for this migration

Reply:
Yes, @ernestotejeda, this migration doesn't grant authorship. Others with a big refactoring might be, but not this one. Please remove it.